### PR TITLE
Improve on demand sandbox & cleanup naming

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Improved the look n feel of the new on-demand sandbox. Added some feature that will be necessary when converting all the pages to this new sandbox

| before | after |
| --- | --- |
| image | image |

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Comment by sallainCoveo** [comment]
It might be unrelevant to this branch, but I've noticed that I'm denied access to the advanced section in the legacy documentation, is it expected?

https://user-images.githubusercontent.com/58052881/204588906-9a16cdf3-11f0-4eb2-81b2-6c8c73b16881.mp4

**Review by sallainCoveo — APPROVED** [review]
The new demos seems to look and work well!

File: packages/website/src/building-blocs/Demo.tsx [hunk 1]
```diff
@@ -25,42 +42,67 @@ const useStyles = createStyles((theme) => ({
...
+        minHeight: 100,
+    },
+    previewWrapper: {
+        padding: center ? undefined : theme.spacing.md,
+        height: grow ? MAX_HEIGHT : '100%',
+    },
+    code: {
+        minHeight: 100,
+        position: 'relative',
+        backgroundColor: '#1E1E1E',
```
[the remark this answers is not in the document]
Comment (gdostie) [reply packages/website/src/building-blocs/Demo.tsx:56]: No, this is a color out of the Coveo theme, it is the color of the vsdark theme coming from prism-react-renderer

**Comment by gdostie** [comment]
> It might be unrelevant to this branch, but I've noticed that I'm denied access to the advanced section in the legacy documentation, is it expected?

Looking at the error from your screen capture, it says somethings about a chrome extension intercepting something. Since its working on my machine, I am suspecting to take a look at your chrome extensions 🕵️

**Pull request merged**